config.go

```diff
@@ -24,6 +24,9 @@ type Config struct {
 	// TODOComment is used in case no default value was defined for a property. It is set by
 	// default in NewConfig but can be emptied to remove the comment altogether.
 	TODOComment string
+
+	// Minimal mode only renders properties which are required and have no default value
+	Minimal bool
```

Comment:
Could you add a `Minimal()` (or whatever you think makes sense) as a functional option

Reply:
Added